src/wp-includes/default-constants.php

```diff
@@ -407,13 +407,15 @@ function wp_templating_constants() {
 	 * Filesystem path to the current active template directory.
 	 *
 	 * @since 1.5.0
+	 * @deprecated 6.4.0 Use `get_template_directory()` instead.
```

Comment:
It's important to improve the documentation for this function by adding a `@see` annotation, similar to how it's done for other `deprecated` functions.

There are two options available, but the majority of other `deprecated` functions use the first option:

1.
```suggestion
	 * @deprecated 6.4.0 Use {@see 'get_template_directory()'} instead.
```

2.
```suggestion
	 * @deprecated 6.4.0 Use get_template_directory() instead.
	 * @see get_template_directory()
```


🔢 If changed, applies elsewhere in the PR.

Reply:
@mukeshpanchal27 Will update to the 2nd version, that's the correct one looking at other code code. We shouldn't use `@see` within the `@deprecated` annotation.

Reply:
Addressed in https://github.com/WordPress/wordpress-develop/pull/5160/commits/03f884431bdae8f49d4aadde6ac9587239a84434